.eslintrc.js

```diff
@@ -31,10 +35,10 @@ module.exports = {
     '@typescript-eslint/no-unused-vars': [
       'error',
       {
-        'args': 'after-used',
-        'argsIgnorePattern': '^_$',
-        'ignoreRestSiblings': true
-      }
+        args: 'after-used',
+        argsIgnorePattern: '^_$',
+        ignoreRestSiblings: true,
+      },
```

Comment:
Not really related but :fire: A U T O F O R M A T T I N G :fire: updated these so thought why not

Reply:
my inline comment was not good : (. That inline commenting thing is kinda hard as it does not do formatting